corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js

```diff
@@ -0,0 +1,55 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("AppSelect.MenuList", function (MenuList, FormplayerFrontend, Backbone, Marionette, $, _) {
+    MenuList.Controller = {
+        selectMenu: function (app_id, select_list) {
+
+            var fetchingApps = FormplayerFrontend.request("app:select:menus", app_id, select_list);
```

Comment:
this is called the same thing as above, but it doesn't look like it's still fetching apps